config/elasticsearch.php

```diff
@@ -3,7 +3,10 @@
 
 return [
     'hosts' => (array) env('ES_HOST', 'localhost:9200'),
-
-    'search_timeout' => get_float(env('ES_SEARCH_TIMEOUT')) ?? 5,
-    'search_connect_timeout' => get_float(env('ES_SEARCH_CONNECT_TIMEOUT')) ?? 0.5,
+    'connectionParams' => [
+        'client' => [
+            'timeout' => get_float(env('ES_SEARCH_TIMEOUT')) ?? 5,
+            'connect_timeout' => get_float(env('ES_SEARCH_CONNECT_TIMEOUT')) ?? 0.5,
```

Comment:
the original idea was to allow longer timeouts for indexing =S

Reply:
also the env name can use an update

Reply:
elasticsearch doesn't wait for indexing to run before responding; responses get returned immediately unless `?refresh=wait_for` is set

Reply:
although it still needs to wait for the parser and errors to output 🤔

Reply:
and can still use name update because it's initially named for search function (`ES_SEARCH_...`)